src/multiselect.html

```diff
@@ -214,6 +215,19 @@
             this._field = this._root.querySelector('.multiselect-field');
             this._popup = this._root.querySelector('.multiselect-popup');
             this._list = this._root.querySelector('.multiselect-list');
+
+            var _this = this;
+            var observer = new MutationObserver(function(mutations) {
+                _this._root = _this.createRootElement();
+                _this._control = _this._root.querySelector('.multiselect');
+                _this._field = _this._root.querySelector('.multiselect-field');
+                _this._popup = _this._root.querySelector('.multiselect-popup');
+                _this._list = _this._root.querySelector('.multiselect-list');
```

Comment:
I believe, init block should be extracted to avoid DRY violation. This will also allow us to get rid of `_this` variable.